docs/algorithms/kem/ntru.yml

```diff
@@ -16,6 +16,9 @@ auxiliary-submitters:
 - Zhenfei Zhang
 crypto-assumption: NTRU in Z[x]/(q, x^n-1) with prime n and power-of-two q
 website: https://ntru.org/
+# TODO: under NTT (Nippon Telegraph and Telephone) consideration?
+standardization-status: Not selected
```

Comment:
So why not write "Not selected by NIST. Under consideration for standardization by NTT" (link available?)?